.github/workflows/docker-publish-indexer.yml

```diff
@@ -2,11 +2,8 @@ name: Docker Build Push (Indexer)
 
 on:
   push:
-    branches: [main, hot-fix, compatibility-breaking-changes, bump-pg-version]
-    # Publish semver tags as releases.
     tags: ["v*.*.*"]
-  pull_request:
-    branches: [main, hot-fix, compatibility-breaking-changes, bump-pg-version]
+  workflow_dispatch:
```

Comment:
Using `workflow_dispatch` trigger allows us to build specified images manually. I think it is better for us. We can build images by specifying a branch name.

Reply:
Does workflow_dispatch require Admin role?

Reply:
I see.
> Write access to the repository is required to perform these steps.